wiki/Source-Code-Organization.md

```diff
@@ -26,8 +26,8 @@ The toolchain consists of components in both Node.js and Maven worlds.
 -   **`packages/node/polyglotpkg`** - Packages Node.js/Python/PowerShell project into Polyglot bundle that can be executed by vRO 8.1+
 -   **`packages/node/vropkg`** - CLI tool and library for transforming vRO content source through multiple forms (`JavaScript source ➡️ XML source ➡️ .package binary`)
 -   **`packages/node/vrotsc`** - allows expressing every type vRO content (workflows, actions, config elements) as TypeScript source code; transpiles TypeScript into vRO-compatible JavaScript and XML ready for consumption by vRO
--   **`packages/node/vro-language-server`** - A Node.js implementation of the [Language Server Protocol (LSP)](https://github.com/Microsoft/language-server-protocol). It analyses type information received from the vRO server and maintains semantic knowledge about a vRO solution implemented in JavaScript. Depends on vRO Hint Plugin that is part of the [vRealize Build Tools](https://flings.vmware.com/vrealize-build-tools) fling. _No longer maintained, still remains for compatibility._
--   **`packages/node/vro-language-server/protocol`** - A set of [Protocol Buffer](https://developers.google.com/protocol-buffers/) message definitions that are used by the vRO language server and the vRO Hint plug-in as a serialization format for communication and storage purposes. _No longer maintained, still remains for compatibility._
+-   **`packages/node/vro-language-server`** - A Node.js implementation of the [Language Server Protocol (LSP)](https://github.com/Microsoft/language-server-protocol). It analyses type information received from the vRO server and maintains semantic knowledge about a vRO solution implemented in JavaScript. The updated version is operational with vRO 8, but not with the vRO 7 anymore. The plugin is not needed for fetching the data from the vRO, but rather only a connection to it is required and a valid maven profile. Moreover, local js files are also analyzed.
+-   **`packages/node/vro-language-server/protocol`** - A set of [Protocol Buffer](https://developers.google.com/protocol-buffers/) message definitions that are used by the vRO language server as a serialization format for communication and storage purposes.
```

Comment:
I think we need to add a bit more documentation on how this is working, maybe a separate md file that explains the communication flow and the components/files involved in this communication.

Also - is this going to work for vRO servers that are using vRA authentication? I think currently all operations for vRDT support only basic auth, which makes most of the things unusable in vra 8.8+ without additional vro configuration?